demo/common/assets.js

```diff
@@ -206,7 +208,9 @@ shakaAssets.UplynkResponseFilter = function(type, response) {
   if (type == shaka.net.NetworkingEngine.RequestType.MANIFEST) {
     // Parse a custom header that contains a value needed to build a proper
     // license server URL
-    shakaAssets.uplynk_prefix = response.headers['x-uplynk-prefix'];
+    if (response.headers['x-uplynk-prefix']) {
```

Comment:
If the header is not present, doesn't this have the side-effect of leaving behind a cached value from a previous manifest?  Is this intentional?  If so, please add a comment explaining why.